Makefile

```diff
@@ -104,5 +109,9 @@ gen_patch_iscp:
 gen_patch_values:
 	diff -u pkg/apis/istio/v1alpha2/values/values_types.pb.go.orig pkg/apis/istio/v1alpha2/values/values_types.pb.go > pkg/apis/istio/v1alpha2/values/fix_values_structs.patch || true
 
-vfsgen:
+vfsgen: data/
+	go get github.com/shurcooL/vfsgen
 	go generate ./cmd/iop.go
+
+iop: vfsgen
```

Comment:
Adding `vfsgen` as a dependency results in a dirty git status as `vfsgen` will always generate the go code with new timestamp. Should we remove this dependency? Unless dev want to make change to the baked in chart, in that case, we need to generate the code with `vfsgen`.

Reply:
Yes, seems like we need to further improve the Makefile and make it smarter. But just like you mentioned, if I remove vfsgen dep, those who work on profiles and charts would say the build is broken. Ideally we should find a way to either `go generate` only when /data changes or exclude vfsgen_data.go from this git repo. Not quite sure, let me investigate more.